src/redux/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+export default configureStore({
+  reducer: {
+    books: booksReducer,
```

Comment:
- [ ] Kindly import the CategoriesReducer and the categories should be added to configureStore method 👍🏿